flink-runtime/src/test/java/org/apache/flink/runtime/io/network/buffer/LocalBufferPoolTest.java

```diff
@@ -65,15 +65,15 @@ class LocalBufferPoolTest {
     private BufferPool localBufferPool;
 
     @RegisterExtension
-    public static final TestExecutorExtension<ExecutorService> EXECUTOR_RESOURCE =
+    static final TestExecutorExtension<ExecutorService> EXECUTOR_EXTENSION =
```

Comment:
Could you clarify a bit on why we change the variable name here?

Reply:
Fixed.